File: samples/server/petstore/jaxrs/src/main/java/io/swagger/api/impl/StoreApiServiceImpl.java
package io.swagger.api.impl;

import io.swagger.api.*;
import io.swagger.api.NotFoundException;
import io.swagger.model.Order;

import javax.ws.rs.core.Response;

public class StoreApiServiceImpl extends StoreApiService {

    @Override
    public Response getInventory()
            throws NotFoundException {
        // do some magic!
        return Response.ok().entity(new ApiResponseMessage(ApiResponseMessage.OK, "magic!")).build();
    }

    @Override
    public Response placeOrder(Order body)
            throws NotFoundException {
        // do some magic!
        return Response.ok().entity(new ApiResponseMessage(ApiResponseMessage.OK, "magic!")).build();
    }

    @Override
    public Response getOrderById(String orderId)
            throws NotFoundException {
        // do some magic!
        return Response.ok().entity(new ApiResponseMessage(ApiResponseMessage.OK, "magic!")).build();
    }

    @Override
    public Response deleteOrder(String orderId)
            throws NotFoundException {
        // do some magic!
        return Response.ok().entity(new ApiResponseMessage(ApiResponseMessage.OK, "magic!")).build();
    }

}

File: samples/openapi3/server/petstore/kotlin-springboot-reactive/src/main/kotlin/org/openapitools/api/StoreApiServiceImpl.kt
package org.openapitools.api

import org.openapitools.model.Order
import kotlinx.coroutines.flow.Flow;
import org.springframework.stereotype.Service
@Service
class StoreApiServiceImpl : StoreApiService {

    override suspend fun deleteOrder(orderId: kotlin.String): Unit {
        TODO("Implement me")
    }

    override suspend fun getInventory(): Map<String, kotlin.Int> {
        TODO("Implement me")
    }

    override suspend fun getOrderById(orderId: kotlin.Long): Order {
        TODO("Implement me")
    }

    override suspend fun placeOrder(order: Order): Order {
        TODO("Implement me")
    }
}
